--- duneprototypes-10_10_02d00/duneprototypes/Protodune/dualphase/CMakeLists.txt	2025-08-29 18:02:31.000000000 -0500
+++ spack-src/duneprototypes/Protodune/dualphase/CMakeLists.txt	2025-09-17 11:56:03.997620513 -0500
@@ -25,7 +25,7 @@
 if(FALSE)
 cet_build_plugin(
 			Purity art::module LIBRARIES
-			dunesim::DetSim
+			
                         lardataobj::RecoBase
                         larreco::Calorimetry
                         larcorealg::Geometry
@@ -53,7 +53,7 @@
 endif()
 cet_build_plugin(
 			DEdxdp art::module LIBRARIES
-			dunesim::DetSim
+			
                         lardataobj::RecoBase
                         larreco::Calorimetry
                         larcorealg::Geometry
--- duneprototypes-10_10_02d00/duneprototypes/Protodune/dualphase/Purity_module.cc	2025-08-29 18:02:31.000000000 -0500
+++ spack-src/duneprototypes/Protodune/dualphase/Purity_module.cc	2025-09-17 11:56:03.996620512 -0500
@@ -37,7 +37,7 @@
 #include "lardataobj/RecoBase/Track.h"
 #include "lardataobj/RecoBase/TrackHitMeta.h"
 #include "larreco/Calorimetry/CalorimetryAlg.h"
-#include "dunesim/DetSim/Service/DPhaseSimChannelExtractService.h"
+
 
 #include "TTree.h"
 #include "TH1.h"
--- duneprototypes-10_10_02d00/duneprototypes/Protodune/singlephase/NearlineMonitor/CMakeLists.txt	2025-08-29 18:02:31.000000000 -0500
+++ spack-src/duneprototypes/Protodune/singlephase/NearlineMonitor/CMakeLists.txt	2025-09-17 11:56:03.998620515 -0500
@@ -108,7 +108,7 @@
               lardataobj::RecoBase
               lardataalg::DetectorInfo
               lardata::headers
-              duneopdet::OpticalDetector
+              
               art::Framework_Core
               art::Framework_Principal
               art_root_io::tfile_support
--- duneprototypes-10_10_02d00/duneprototypes/Protodune/singlephase/NearlineMonitor/PlotOpticalDetails_module.cc	2025-08-29 18:02:31.000000000 -0500
+++ spack-src/duneprototypes/Protodune/singlephase/NearlineMonitor/PlotOpticalDetails_module.cc	2025-09-17 11:56:03.999620517 -0500
@@ -27,7 +27,7 @@
 // LArSoft includes
 #include "lardataobj/RecoBase/OpFlash.h"
 #include "lardataobj/RecoBase/OpHit.h"
-#include "duneopdet/OpticalDetector/OpFlashSort.h"
+
 
 // ART includes.
 #include "art/Framework/Core/EDAnalyzer.h"
--- duneprototypes-10_10_02d00/duneprototypes/Protodune/singlephase/CRT/data/CMakeLists.txt	2025-08-29 18:02:31.000000000 -0500
+++ spack-src/duneprototypes/Protodune/singlephase/CRT/data/CMakeLists.txt	2025-09-17 12:41:14.935126839 -0500
@@ -16,6 +16,7 @@
             DICT_LIBRARIES
                         lardataobj::Simulation
                         nusimdata::SimulationBase
+			larcorealg::Geometry
 
 )
 
--- duneprototypes-10_10_02d00/CMakeLists.txt	2025-08-29 18:02:31.000000000 -0500
+++ spack-src/CMakeLists.txt	2025-09-17 11:56:03.998620515 -0500
@@ -43,9 +43,9 @@
 find_package( larana REQUIRED EXPORT )
 find_package( larevt REQUIRED EXPORT )
 find_package( larsim REQUIRED EXPORT )
-find_package( duneopdet REQUIRED EXPORT )
+
 find_package( dunecalib REQUIRED EXPORT )
-find_package( dunesim REQUIRED EXPORT )
+
 find_package( nusimdata REQUIRED EXPORT )
 find_package( nug4 REQUIRED EXPORT )
 find_package( dunepdlegacy REQUIRED EXPORT )
